pylint/checkers/imports.py

```diff
@@ -191,9 +191,9 @@ def _make_graph(
         "Used when `from module import *` is detected.",
     ),
     "W0402": (
-        "Uses of a deprecated module %r",
+        "Deprecated module %r",
```

Comment:
We will probably need to update some tests as we test for the message of a warning and that changes with this PR.
You can do this by using the auto-updates (the failing tests will show how to do this) or by simplying searching for `Uses of a deprecated module` in our codebase and replacing it with the new message.

Reply:
this is resolved by [d14785a](https://github.com/PyCQA/pylint/pull/6387/commits/d14785a5a35b070f55c17faf2c78a3d6b7296997)

now `python ./tests/test_functional.py` shows 726 passed, 24 skipped (why are some tests being skipped?)

Reply:
oh hold on, there are more failing tests in other scripts (is there an obvious way to run them all?)

Reply:
I believe `python tests/test_functional.py --update-functional-output -k "test_functional"` should update them all at once.

Skipped tests are dependent on your environment. Some tests are only run on certain python version or operating systems.